mp/src/game/client/momentum/ui/controls/ImageGallery.cpp

```diff
@@ -342,6 +342,11 @@ void ImageGallery::Paint()
     }
 }
 
+void ImageGallery::PerformLayout() 
+{ 
+    UpdateButtonStates(); 
```

Comment:
Ensure you call `BaseClass::PerformLayout()`

Reply:
Oh right, that's fixed!